UCSRest/src/test/resources/application.properties

```diff
@@ -1,7 +1,8 @@
 server.port=9998
 
 ucs.core.uri=http://localhost:9998
-ucs.core.journal-dir=/tmp/ucf
+ucs.core.journal-path=/tmp/ucf

```

Comment:
funny characters at the end of each line in this file

Reply:
yeah this is a carriage-return character probably generated by windows. use vim and then :s/^M$//